src/config/plugin.ts

```diff
@@ -83,7 +83,7 @@ async function findRoot(name: string | undefined, root: string) {
   if (name) {
     let pkgPath
     try {
-      pkgPath = resolvePackage(name, {paths: [__dirname, root]})
+      pkgPath = resolvePackage(name, {paths: [root]})
```

Comment:
@mdonnalley why is __dirname being removed?

Reply:
@peternhale two reasons: it's redundant and it was causing linked plugins to resolve to the version in node_modules - not your local version.

In the case of an unlinked plugin, we have
`__dirname` => `/Users/mdonnalley/.nvm/versions/node/v17.1.0/lib/node_modules/@salesforce/cli/node_modules/@oclif/core/lib/config`
`root` => `/Users/mdonnalley/.nvm/versions/node/v17.1.0/lib/node_modules/@salesforce/cli`

In this case, `__dirname` is redundant since it's already included by the `root` path

In the case of a linked plugin, we have

`__dirname` => `/Users/mdonnalley/.nvm/versions/node/v17.1.0/lib/node_modules/@salesforce/cli/node_modules/@oclif/core/lib/config`
`root` => `/Users/mdonnalley/repos/oclif/plugin-version`

In this case, `__dirname` was causing the resolution to return the node_modules version instead of the local version

In both cases, `root` is sufficient for resolving the plugin's location